src/main/java/Start.java

```diff
@@ -0,0 +1,35 @@
+import employeeServiceCRUD.controller.AddressController;
+import employeeServiceCRUD.controller.DepartmentController;
+import employeeServiceCRUD.controller.EmployeeController;
+import employeeServiceCRUD.controller.ModelController;
+import employeeServiceCRUD.repositories.AddressRepositoryIml;
+import employeeServiceCRUD.repositories.DepartmentRepositoryIml;
+import employeeServiceCRUD.repositories.EmployeeRepositoryIml;
+import employeeServiceCRUD.service.AddressService;
+import employeeServiceCRUD.service.DepartmentService;
+import employeeServiceCRUD.service.EmployeeService;
+
+import java.sql.SQLException;
+
+public class Start {
```

Comment:
[minor] name for this class can be EmplyeeServiceApplication, you will see such names when learn Spring